quicktry/quicktry.py

```diff
@@ -18,14 +36,15 @@ def query_images():
     return tags
 
 
-def execute(workdir, data, stdin):
+def execute(workdir, data, stdin, language):
     # create the client to the docker service
     cli = Client(base_url='unix://var/run/docker.sock', version='auto')
 
     # generate the temporary path for the worker
     with tempfile.TemporaryDirectory(dir=workdir) as dirpath:
         # create the input file
-        with open(os.path.join(dirpath, 'input.py'), 'w') as f:
+        input_file="input" + lang_config[language]["ext"]
+        with open(os.path.join(dirpath, inpute_file), 'w') as f:
```

Comment:
Use `"{}".format` instead of +

Typo on line 47 should be corrected to `input_file`